### ISSUE TITLE
Investigate using CrowdIn for the translations platform

### ISSUE THREAD

**Issue by chenxiaolong** [chenxiaolong/DualBootPatcher#30]
Following from discussion in issue #27.

**Comment by Caio99BR** [comment]
The crowdin support the 30 most popular languages, but this is only a trial for 10 days.
We can download all strings but this not in android folder format (values-pt-rBR), i think we can get from another app and make this manually (and up too)

**Comment by chenxiaolong** [comment]
Do you know if that also applies to the free open-source plan that they have? https://crowdin.com/page/open-source-project-setup-request

**Comment by Caio99BR** [comment]
Yes this is applied to open source plan, you will just need to create the project in crowdin and request this.
More open source of this project is basically impossible, i thing :P

**Comment by chenxiaolong** [comment]
I've just requested the open-source plan :P I'll start taking a look at the command line tools to see it it can convert to/from the Android strings.xml format.

I'll also start looking into automatically merging translations nightly. I don't have this set up currently and I think it would be a very useful addition :)

**Comment by SerenityS** [comment]
Great. I want to participate in this project.

**Comment by chenxiaolong** [comment]
Got the approval! I'll hopefully have some time to set everything up this weekend.

**Comment by Caio99BR** [comment]
Any news?

**Comment by Caio99BR** [comment]
Change it to Milestone 10 and also refresh translations from Transifex

**Comment by chenxiaolong** [comment]
> Change it to Milestone 10 and also refresh translations from Transifex

Agreed and done. Commit: 7c45ace6195afe92cbd3d87fc9820f5dc7e876fe

**Comment by Caio99BR** [comment]
Another refresh on Transifex, can you update it?

**Comment by chenxiaolong** [comment]
I've set up the build server to upload the original `strings.xml` to Transifex after each successful build of the master branch. I'll be setting up a daily job to pull the new translations from Transifex also.

I checked out CrowdIn, but I probably won't switch until they improve their command-line tools, which are not very good compared to Transifex.

**Comment by Caio99BR** [comment]
Ok, so, remove the Milestone and close it :)
